rewrite-maven/src/test/kotlin/org/openrewrite/maven/RemoveUnneededDependencyOverridesTest.kt

```diff
@@ -0,0 +1,887 @@
+/*
+ * Copyright 2020 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.maven
+
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import kotlin.io.writeText
+import kotlin.text.trimIndent
+
+class RemoveUnneededDependencyOverridesTest : MavenRecipeTest {
+
+    @Test
+    fun matchesParentDM_remove() {
```

Comment:
```suggestion
    fun matchesParentDmRemove() {
```
underscores, in a JVM-based language? Whaddaya think this is, jython?

Reply:
JRuby ;)

I'm happy to conform to whatever our style is. For unit tests specifically in JUnit, I've used underscores to split "conditions" before since you can't use a "given when then" or sentence-based syntax (a la BDD).

Reply:
Updated in the newest commit.